source/admin_manual/importing_mailboxes.rst

```diff
@@ -55,3 +55,81 @@ If you have only read privileges, you can try using in-memory indexes.
 
   doveadm import -u destuser maildir:/opt/backup/destuser/Maildir:INDEX=MEMORY "" ALL
 
+
+Merging Storages
+----------------
+
+In some disaster recovery cases you may end up having mails for the same user
+in two different locations, and need to merge them. For example the storage
+goes down and fixing it takes a long time, so during the fixing you can let the
+users access their emails as an empty account, which can receive new mails.
+Later on you can use ``doveadm import`` to merge the mailboxes.
+
+Note that there is no way to make this solution perfect:
+ * IMAP clients that have cached mails locally will delete their local caches
+   and have to re-download mails later on.
+ * POP3 clients that leave mails on server will notice all the mails are gone,
+   and delete their local UIDL caches. When old mails come back, they're
+   re-downloaded as new emails (duplicates).
+
+Also, there are 3 alternative ways of how mails can be imported into mailboxes:
```

Comment:
I wonder whether this should be subsections and not a listing, because further down the "Example for the 2nd case where [...]" might be a bit confusing if not everything was read sequentially from top down (which I didn't do).

Reply:
I think it would be more confusing to read the alternatives if they're spread out all over the place. But I changed it now to a numbered list and added `2nd case ("New mails are imported on top of the old recovered mails")` - maybe that's enough?